tools/netboxr/netboxr_xml.xml

```diff
@@ -0,0 +1,182 @@
+<tool id="netboxr_tool" name="NetBox" version="1.6.0">
+<!-- A simple description of the tool that will appear in the tool
+panel in Galaxy. -->
+<description> netboxr enables automated discovery of biological process
+modules by network analysis.</description>
+<!-- Handles exit codes in Galaxy. -->
+<requirements>
+<requirement type="package">r-optparse</requirement>
+<requirement type="package" version="1.6.0">bioconductor-netboxr</requirement>
+</requirements>
+<stdio>
```

Comment:
remove stdio and use detect_errors see https://docs.galaxyproject.org/en/latest/dev/schema.html#error-detection